From 460264b2f97cb8aa71d989b6753f77b295c62f08 Mon Sep 17 00:00:00 2001
From: Marie-Cecile Afantenos <marie.cecile.afantenos@intel.com>
Date: Tue, 25 Jun 2013 10:00:06 +0200
Subject: [PDK MR2 ABSP] generate MR2 recovery fstab format

BZ: 118706

exchange position of block to be conform with the fsmgr lib

Change-Id: Iaa14959b1963a32ab086caab8564dabc23e74103
Signed-off-by: Marie-Cecile Afantenos <marie.cecile.afantenos@intel.com>
---
 partition.py |    4 +++-
 1 files changed, 3 insertions(+), 1 deletions(-)

diff --git a/partition.py b/partition.py
index fa06f3d..e180c9e 100755
--- a/partition.py
+++ b/partition.py
@@ -258,11 +258,13 @@ def generate_mbr_recovery_fstab_file(storage, global_data):
 
         current_line.append("#size_hint=%d" % (partition["size"]))
         current_line.append("\n")
+        current_line.append(global_data["partition_format"] % (storage["base_name"], partition_id))
+        current_line.append("\t")
         current_line.append("/%s" % (partition_name))
         current_line.append("\t")
         current_line.append("%s" % (partition["fs_type"]))
         current_line.append("\t")
-        current_line.append(global_data["partition_format"] % (storage["base_name"], partition_id))
+        current_line.append("%s" % (partition["mount_options"]))
         current_line.append("\t")
         current_line.append("length=%d" % (partition["length"]))
         current_line.append("\n")
-- 
1.7.4.1

